src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -237,6 +237,12 @@ public string PathPrefix
         /// </summary>
         public bool RequireRestrictionAnnotationsToGenerateComplexPropertyPaths { get; set; } = true;
 
+        /// <summary>
+        /// Gets/Sets a value indicating whether or not to use the HTTP status code class of 2XX
+        /// to represent all response codes between 200 - 299.
+        /// </summary>
+        public bool UseHTTPStatusCodeClass2XX { get; set; } = false;
```

Comment:
Consider naming this to something like `UseSuccessStatusCodeRange` to make it implementation agnostic.

Reply:
Aah... I see. I will implement your suggestion.